src/mds/cephfs_features.h

```diff
@@ -48,6 +49,7 @@
   CEPHFS_FEATURE_NAUTILUS,              \
   CEPHFS_FEATURE_DELEG_INO,             \
   CEPHFS_FEATURE_OCTOPUS,               \
+  CEPHFS_FEATURE_MAX,                   \
```

Comment:
This name is not idiomatic within our codebase; a _MAX is used to indicate the lowest unused value and is not itself encoded anywhere.